server/prisma/schema.prisma

```diff
@@ -281,27 +279,28 @@ model contentViews {
   contentId Bytes    @db.Binary(16)
   date      DateTime @default(dbgenerated("(curdate())")) @db.Date
   userId    Bytes    @db.Binary(16)
-  content   content  @relation(fields: [contentId], references: [id])
+  content   content  @relation(fields: [contentId], references: [id], onDelete: Cascade, onUpdate: NoAction)
 
   @@unique([date, contentId, userId])
 }
 
 model users {
-  userId             Bytes                @id @default(dbgenerated("(uuid_to_bin(uuid(),1))")) @db.Binary(16)
-  email              String               @unique
+  userId             Bytes                  @id @default(dbgenerated("(uuid_to_bin(uuid(),1))")) @db.Binary(16)
+  email              String                 @unique
   firstNames         String?
   lastNames          String
-  isAdmin            Boolean              @default(false)
-  isLibrary          Boolean              @default(false)
-  isAnonymous        Boolean              @default(false)
-  isAuthor           Boolean              @default(false)
+  isAdmin            Boolean                @default(false)
```

Comment:
I just realized we need another type of admin that isn't an editor, which is the ability to change settings on the server itself. For example, one new feature we need to add soon is the ability to maintain the `doenetmlVersions` table when we update the DoenetML code. We don't want all our editors to have that ability.

How about we change the name of this permission to `isEditor`? Once you've renamed all "Admin" to "Editor" in the code codebase, then go ahead and add `isAdmin` back into this table so that we have both `isEditor` and `isAdmin`, since we know we'll need `isAdmin`, too.